src/components/app-configure/access/DidDelegate.vue

```diff
@@ -0,0 +1,349 @@
+<script lang="ts" setup>
+import moment from 'moment'
+
+import SettingCard from '@/components/app-configure/SettingCard.vue'
+import VCard from '@/components/lib/VCard/VCard.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import { useAppsStore } from '@/stores/apps.store'
+import { useAppId } from '@/use/getAppId'
+import { truncate } from '@/utils/stringUtils'
+
+const appId = useAppId()
+const appsStore = useAppsStore()
+
+const app = appsStore.app(appId)
+const delegates = app.access.delegates
+
+function addDelegate() {
+  //
+}
+</script>
+
+<template>
+  <section name="did-delegate">
+    <SettingCard>
+      <template #title>Delegates</template>
+      <template #controls>
+        <img
+          src="@/assets/iconography/plus.svg"
+          class="cursor-pointer control-icons"
+          @click.stop="addDelegate"
+        />
+      </template>
+      <template #description>
+        <VStack
+          gap="2rem"
+          justify="space-between"
+          sm-direction="column"
+          md-direction="column"
+        >
+          <div class="description">
+            Delegations can be used to have assigned delegate servers manage the
+            DID’s on behalf of the users. Manage the delegates that you have
+            assigned the ability to perform actions on behalf of your users.
+          </div>
+          <VStack gap="2rem">
+            <a href="#">
+              <VStack
+                direction="column"
+                sm-direction="row"
+                md-direction="row"
+                gap="0.5rem"
+                align="center"
+              >
+                <img
+                  src="@/assets/iconography/view-docs.svg"
+                  class="help-action-icons"
+                />
+                <span class="help-title">View Docs</span>
+              </VStack>
+            </a>
+            <a href="#">
+              <VStack
+                direction="column"
+                sm-direction="row"
+                md-direction="row"
+                gap="0.5rem"
+                align="center"
+              >
+                <img
+                  src="@/assets/iconography/play-video.svg"
+                  class="help-action-icons"
+                />
+                <span class="help-title">View Guide</span>
+              </VStack>
+            </a>
+          </VStack>
+        </VStack>
+      </template>
+      <VCard variant="depressed" class="delegates-card">
+        <div class="table-container">
+          <div v-if="delegates.length" class="table">
```

Comment:
Avoid implicit coercion in conditionals. 

```suggestion
          <div v-if="delegates.length === 0" class="table">
```

Reply:
Done here https://github.com/arcana-network/developer-dashboard/pull/144/commits/97ff4b0581520d7855f298507c9af5bf129ab3ba